src/napari_cryoet_data_portal/_sample_data.py

```diff
@@ -22,20 +19,32 @@ def tomogram_10000_ts_027() -> List[FullLayerData]:
 
 def _read_tomogram_from_10000(name: str) -> List[FullLayerData]:
     client = Client()
-    
+
     tomogram_spacing_url = f"https://files.cryoetdataportal.cziscience.com/10000/{name}/Tomograms/VoxelSpacing13.480/"
-    tomogram_spacing = next(TomogramVoxelSpacing.find(client, [TomogramVoxelSpacing.https_prefix == tomogram_spacing_url]))
+    tomogram_spacing = TomogramVoxelSpacing.find(
+        client, [TomogramVoxelSpacing.https_prefix == tomogram_spacing_url]
+    ).pop()
 
-    tomogram: Tomogram = next(tomogram_spacing.tomograms)
+    tomogram: Tomogram = tomogram_spacing.tomograms.pop()
```

Comment:
`[0]` would be equivalent, but I assume we only expect one element, so it doesn't really matter.